### DISCUSSION TITLE
Manual NTP server ip address in network config

### DISCUSSION THREAD

**Discussion by pH03NIxM** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#535] [General]
Welcome! Thanks for the great board, it is working fine :) My board is running on a public fixed ip address to access it from remote. I noticed when i blocked the network traffic (to access it from specific IP range), that it need to access to NTP servers during boot, othercase i get E13 error (pump 54144, 4 wire one). 
I have local ntp server (for my other devices), so it would be fine to set up NTP server manually in the network config. 
Thanks!

**Comment by visualapproach** [comment]
Noted, thanks

**Reply by PFalko** [reply]
I second this, e.g. try using the router as ntp server address and/or a manual entry.

**Reply by visualapproach** [reply]
Feature added in beta version. Update via web UI to get both the new firmware and the updated html file. In network config, reenter the fields and save. Note that if you had a static IP set, there will be weird looking text the first time after update. Like IP: [192,168,0,1]. Just overwrite it with the normal layout: 192.168.0.1 and so on.

Let me know if it works or not. (tested it and it appears to work for me)

**Reply by pH03NIxM** [reply]
Great! Just tried, and working fine, thanks!
